packages/dd-trace/src/runtime_metrics/runtime_metrics.js

```diff
@@ -34,15 +34,27 @@ reset()
 const runtimeMetrics = module.exports = {
   start (config) {
     const clientConfig = DogStatsDClient.generateClientConfig(config)
+    const watchers = []
 
-    try {
-      nativeMetrics = require('@datadog/native-metrics')
+    if (config.runtimeMetrics.gc.enabled !== false) {
+      const gcCollector = config.runtimeMetrics.gc.collector || 'default'
 
-      if (hasGCObserver) {
-        nativeMetrics.start('loop') // Only add event loop watcher and not GC.
+      if (hasGCProfiler && (gcCollector === 'profiler' || gcCollector === 'default')) {
+        startGCProfiler()
+      } else if (hasGCObserver && (gcCollector === 'observer' || gcCollector === 'default')) {
+        startGCObserver()
       } else {
-        nativeMetrics.start()
+        watchers.push('gc')
       }
```

Comment:
```suggestion
      if (hasGCProfiler) {
        if (gcCollector === 'profiler' || gcCollector === 'default') {
          startGCProfiler()
        } else {
          startGCObserver()
        }
      } else {
        watchers.push('gc')
      }
```

The condition checked in two different spots for default. That is definitely off.